Error when adding awesome-grid schematic to an empty project

### Description  
Describe the issue.

- Ignite UI CLI version: 5.0.0-rc.1
- Framework: Angular
- Project type: ig-ts|igx-ts

### Steps to reproduce  

1. Create a new empty project `ng new newAngularProject --collection="@igniteui/angular-schematics" --template=side-nav
`
2. Add and awesome-grid `ng g @igniteui/angular-schematics:c awesome-grid newAwesomeGrid
`
3. Start the project `ng serve -o`

### Result  
The project doesn't start and there's an error message displayed in the terminal:
ERROR in src/app/app.module.ts:11:60 - error TS2307: Cannot find module 'igniteui-angular-charts'.

11 import { IgxSparklineModule, IgxSparklineCoreModule } from 'igniteui-angular-charts';

### Expected result  
The project should start without throwing any errors.

--- a/packages/igx-templates/igx-ts/custom-templates/awesome-grid/index.ts
+++ b/packages/igx-templates/igx-ts/custom-templates/awesome-grid/index.ts
@@ -32,6 +32,7 @@ class IgxGridAwesomeTemplate extends IgniteUIForAngularTemplate {
 			},
 			{ import: "FormsModule", from: "@angular/forms" }
 		];
+		this.packages = ["tslib@^1.7.1", "igniteui-angular-core@~8.2.12", "igniteui-angular-charts@~8.2.12"];
 	}
 }
 module.exports = new IgxGridAwesomeTemplate();

